### DISCUSSION TITLE
How to measure drag and lift in a cylindrical flow case？

### DISCUSSION THREAD

**Discussion by duanlinlim** [DualSPHysics/DualSPHysics#272] [DUALSPHYSICS_V5.2]
%computeforces% -dirin %diroutdata% -onlymk:10 -viscolam:0.001 -savecsv %dirout2%/_ForceCylinder_DBC         What kind of lateral force is this？

**Comment by alexbexe** [comment]
You are creating a file with forces in X,Y,Z and magnitude
